### ISSUE TITLE
TaskbarX loop crash

### ISSUE THREAD

**Issue by Audiozz** [ChrisAnd1998/TaskbarX#437]
TaskbarX won't stop opening and closing at self to the point it lags my computer.

I am pretty sure this error: `@Looper1 | Error: Missing method 'instance object [TaskbarX] Accessibility.IAccessible::get_accRole(object)' from class 'System.Int32'.`
Have to do with the crashing.

**Comment by ChrisAnd1998** [comment]
@Audiozz I've got some other reports about this specific error. The problem is that i'm unable to reproduce this. I can only fix this once i know where the issue comes from exactly and manage to come up a alternative. You can help me pin point the issue by going here: https://github.com/ChrisAnd1998/TaskbarX/issues/274 and start the **How can i debug?** section. Hopefully this shows me where i need to look exactly.

**Comment by ChrisAnd1998** [comment]
@Audiozz Can you try the latest Pre-Release 1.6.3.0 both x32 and x64 ?

**Comment by Northrook** [comment]
> @Audiozz Can you try the latest Pre-Release 1.6.3.0 both x32 and x64 ?

I just gave the 1.6.3.0 x64 version a try, and it fixed the issue for me.

The issue started seemingly out of the blue for me on version 1.6.1.0, after readjusting monitor settings. Updating to 1.6.2.0 did not fix the issue. Moving to 1.6.3.0 did.

**Comment by Audiozz** [comment]
> @Audiozz Can you try the latest Pre-Release 1.6.3.0 both x32 and x64 ?

I did right now both worked perfectly thank you.

**Comment by Audiozz** [comment]
> @Audiozz Can you try the latest Pre-Release 1.6.3.0 both x32 and x64 ?

I have another problem but I am not sure if its taskbarx related or not. I can't really explain it in words but here is a picture: 
![Screenshot 2020-11-23 171544](https://user-images.githubusercontent.com/53666724/99979245-8b07c100-2daf-11eb-8e05-a15d37d2533c.png)

**Comment by ChrisAnd1998** [comment]
@Audiozz that's weird XD. That can't be TaskbarX related. I see you have a custom startmenu button maybe the software that does that causes the issue ¯\_(ツ)_/¯

**Comment by Audiozz** [comment]
> @Audiozz that's weird XD. That can't be TaskbarX related. I see you have a custom startmenu button maybe the software that does that causes the issue ¯_(ツ)_/¯

Yeah I deleted it as well.. deleted every thing that was related to the taskbar.. the problem is that's its only happening with the always, hide levels.. I searched all around the internet, couldn't find anything -_-
well anyways thx

**Comment by Liquidream** [comment]
Hiya,

Firstly, big fan of the TaskbarX - I purchased it from the store (last year?) as soon as I saw what it was capable of.
But alas, I - like several others in this thread (and others #591, #502, #406) are now finding it difficult to use on a daily basis as it seems random whether it'll work.

I also seem to suffer regularly from the "Missing method" issue below (even when trying just the Github version):
```
Waiting for Shell_TrayWnd
@Looper1 | Error: Missing method 'instance object [TaskbarX] Accessibility.IAccessible::get_accRole(object)' from class 'System.Int32'.
```

Has there been any progress on this, with the various debug logs you've received?

Thanks in advance.
